BioArchLinux/vsearch/PKGBUILD

```diff
@@ -0,0 +1,32 @@
+# Maintainer: Martin Larralde <martin.larralde@embl.de>
+
+pkgname=vsearch
+pkgver=2.25.0
+pkgrel=0
+pkgdesc="Versatile open-source tool for microbiome analysis. https://doi.org/10.7717/peerj.2584"
+arch=('x86_64' 'aarch64' 'ppc64le')
+url="https://github.com/torognes/vsearch"
+license=('BSD-2-Clause OR GPL-3.0-only WITH Autoconf-exception-3.0')
```

Comment:
According to upstream `README.md` the Autoconf exception applies only to the autoconf files, which are not distributed in the built binary package, so I think the exception should be removed from here.